libafl/src/inputs/encoded.rs

```diff
@@ -79,7 +79,7 @@ where
     T: Tokenizer,
 {
     fn encode(&mut self, bytes: &[u8], tokenizer: &mut T) -> Result<EncodedInput, Error> {
-        let mut codes = vec![];
+        let mut codes: Vec<u32> = vec![];
```

Comment:
you could use `Vec::with_capacity(tokens.len())` for a bit of potential speedup, here